tests/appsec/test_customconf.py

```diff
@@ -81,7 +81,7 @@ def test_requests(self):
     def test_log(self):
         # Check if it's implemented for the weblog variant
         if context.library == "java":
-            stdout.assert_presence("AppSec is FULLY_ENABLED with powerwaf")
+            stdout.assert_presence("AppSec is FULLY_ENABLED with ddwaf")
```

Comment:
```suggestion
            stdout.assert_presence("AppSec is FULLY_ENABLED with")
```

Let's match this partially so that it continues working with old and new versions.

Reply:
it's a regex, so you can do this : 

```suggestion
            stdout.assert_presence(r"AppSec is FULLY_ENABLED with (ddwaf|powewaf)")
```